safe/report/extractors/aggregate_result.py

```diff
@@ -57,6 +57,15 @@ def aggregation_result_extractor(impact_report, component_metadata):
     aggregation_summary_fields = aggregation_summary.keywords[
         'inasafe_fields']
     debug_mode = impact_report.impact_function.debug_mode
+    use_aggregation = bool(impact_report.impact_function.provenance[
```

Comment:
__[flake8]__
* [F841] local variable 'use_aggregation' is assigned to but never used

<!-- Comment by [SideCI](https://sideci.com) -->